source/central/dvla/index.html.md.erb

```diff
@@ -0,0 +1,11 @@
+---
+title: DVLA APIs
+weight: 50
+---
+
+# DVLA APIs
+
+The API catalogue contains the following DVLA APIs:
+
+- [DVLA Vehicle Enquiry Service](Link TBC)
+- [DVLA Authentication](Link TBC]
```

Comment:
What do you need to know for these two links?

Reply:
I'm having trouble creating the proper file structure for the DVLA pages. This is something we can chat about and then we can add the links.